tests/ui/entry_unfixable.rs

```diff
@@ -70,4 +34,17 @@ fn insert_from_different_map2<K: Eq + Hash, V>(m: &mut HashMap<K, V>, n: &mut Ha
     }
 }
 
+fn insert_in_macro<K: Eq + Hash, V>(m: &mut HashMap<K, V>, k: K, v: V) {
+    if !m.contains_key(&k) {
+        m!(m, k, v);
+    }
+}
+
+fn use_map_then_insert<K: Eq + Hash, V>(m: &mut HashMap<K, V>, k: K, v: V) {
+    if !m.contains_key(&k) {
+        let _ = m.len();
+        m.insert(k, v);
+    }
+}
```

Comment:
I think the `_unfixable.rs` suffix of file name mean that suggestions in tests shouldn't be applied mechanically. It doesn't mean tests shouldn't trigger the lint.

It would be better to move these tests to other file and include the tests which suggestions shouldn't be applied mechanically.